File: client/src/app/request/SFN040215.req.ts
import { ShippingDestinationDetailJson } from '../model/core/ShippingDestination.model';
import { CustomerJson } from '../model/core/Customer.model';

/**
 * POST /SFN040215/:customerCode/:shippingDestinationId 届け先保存の要求電文
 *
 * 対応する Java 側クラスは vn.vnext.sefuri.sf.json.SFN0402.response.SFN040215Req
 */
export interface SFN040215ReqJson {
    /** 届け先詳細 */
    destination: ShippingDestinationDetailJson;
}

File: app/vn/vnext/sefuri/sf/json/SFN0402/request/SFN040215Req.java
package vn.vnext.sefuri.sf.json.SFN0402.request;

import com.fasterxml.jackson.annotation.JsonProperty;

import vn.vnext.sefuri.sf.json.common.AbstractJson;
import vn.vnext.sefuri.sf.json.core.ShippingDestinationDetailJson;

/**
 * POST /SFN040215/:customerCode/:shippingDestinationId の要求
 *
 * TypeScript は client/src/app/response/SFN040215.req.ts
 */
public class SFN040215Req extends AbstractJson {

    /** 届け先詳細 */
    @JsonProperty("destination")
    public ShippingDestinationDetailJson destination;


    /** @return 届け先詳細 */
    public ShippingDestinationDetailJson getDestination() {
        return destination;
    }

    /** @param destination 届け先詳細 */
    public void setDestination(ShippingDestinationDetailJson destination) {
        this.destination = destination;
    }
}
